src/test-e2e/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
just curious..what's this do?

Reply:
and why in test-e2e?

Reply:
It is used with Mockito 2 to allow you to test non-open classes as you might hope to (without having to artificially open them or artificially create an interface, etc.).

I accidentally pasted it into test-e2e and forgot to delete.  We’d never use Mockito there.  Oops.